integration-cli/docker_cli_run_test.go

```diff
@@ -940,7 +940,7 @@ func (s *DockerSuite) TestRunDnsDefaultOptions(c *check.C) {
 
 func (s *DockerSuite) TestRunDnsOptions(c *check.C) {
 	testRequires(c, DaemonIsLinux)
-	out, stderr, _ := dockerCmdWithStdoutStderr(c, "run", "--dns=127.0.0.1", "--dns-search=mydomain", "busybox", "cat", "/etc/resolv.conf")
+	out, stderr, _ := dockerCmdWithStdoutStderr(c, "run", "--dns=127.0.0.1", "--dns-search=mydomain", "--dns-opt=ndots:9", "busybox", "cat", "/etc/resolv.conf")
```

Comment:
Just looking at this; do we need a test;

- using *multiple* `--dns-opt` options?
- a test setting these options on the *daemon* (e.g., similar to https://github.com/docker/docker/blob/master/integration-cli/docker_cli_daemon_test.go#L1146), and overriding them on a container?